ios/app/MBXAppDelegate.m

```diff
@@ -1,11 +1,27 @@
 #import "MBXAppDelegate.h"
 #import "MBXViewController.h"
+#import <mbgl/ios/MapboxGL.h>
 #import <mbgl/ios/MGLMapboxEvents.h>
 
 @implementation MBXAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
+    // Set Access Token
+    NSString *accessToken = [[NSProcessInfo processInfo] environment][@"MAPBOX_ACCESS_TOKEN"];
+    if (accessToken) {
+        // Store to preferences so that we can launch the app later on without having to specify
+        // token.
+        [[NSUserDefaults standardUserDefaults] setObject:accessToken forKey:@"access_token"];
+    } else {
+        // Try to retrieve from preferences, maybe we've stored them there previously and can reuse
+        // the token.
+        accessToken = [[NSUserDefaults standardUserDefaults] objectForKey:@"access_token"];
+    }
+    if ( ! accessToken) NSLog(@"No access token set. Mapbox vector tiles won't work.");
+
+    [MapboxGL sharedInstanceWithAccessToken:accessToken];
```

Comment:
It looks like you’re creating a `MapboxGL` and throwing it away. Someone might come along later and delete the line, thinking it does nothing, when in fact you’re setting the access token globally by side effect. Can we call this class method `-setAccessToken:` or `-registerAccessToken:` instead?